src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -1025,6 +1026,12 @@ struct SmartAction
             uint32 wp6;
         } closestWaypointFromList;
 
+        struct
+        {
+            uint32 sound[5];
```

Comment:
This PR has been merged. All those suggestions were done in https://github.com/TrinityCore/TrinityCore/commit/77087db79318443fff1b6fe3c91b55a315c5bf92